commitlint.config.js

```diff
@@ -12,7 +12,7 @@ const Configuration = {
     'subject-full-stop': [2, 'never', '.'],
     'type-case': [2, 'always', 'lower-case'],
     'type-empty': [2, 'never'],
-    'type-enum': [
+    'type-enum': [2, 'always', [
```

Comment:
Revert please, if it is a bug then it needs to be fixed in `webpack-defaults` instead. Not sure actually, seeing this commit messages :D :P https://github.com/webpack-contrib/extract-text-webpack-plugin/pull/728/commits. `git commit -m '...' --no-verify`